### ISSUE TITLE
How to initiate network using torch.nn.init.kaiming_normal?

### ISSUE THREAD

**Issue by cvlearn913** [Project-MONAI/tutorials#558]
When I try to initiate network like this
![image](https://user-images.githubusercontent.com/59678550/154655324-1db5a746-afe5-4e17-aeae-265a4e60c8b6.png)

I got the error as below:
![image](https://user-images.githubusercontent.com/59678550/154655668-216debe6-b0f9-4924-b3ba-58d35c56df0b.png)

How to initiate network correctly？
Thanks in advance !

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

Could you please help check this question?

Thanks in advance.

**Comment by cvlearn913** [comment]
To reproduce the problem,the code is at below
`import torch.nn.init

from monai.networks import normal_init
from monai.networks.layers import Norm
from monai.networks.nets import UNet

model = UNet(
            spatial_dims=3,
            in_channels=1,
            out_channels=3,
            channels=(16, 32, 64, 128, 256),
            strides=(2, 2, 2, 2),
            num_res_units=2,
            norm=Norm.INSTANCE,
        )


model.apply(normal_init(normal_func=torch.nn.init.kaiming_normal))`

**Comment by cvlearn913** [comment]
I got it ，it based on nn.Module.apply（）https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module

**Comment by yiheng-wang-nv** [comment]
Hi @cvlearn913 , sorry for late reply. Did you solve the issue? I checked the source code of `normal_init` and found that it has some limitations, like for this line:
```
normal_func(m.weight.data, 0.0, std)
```
Functions in https://pytorch.org/docs/stable/nn.init.html have different arguments, thus `normal_func` should be able to achieve alterable input. Let me submit a PR to enhance it.

**Comment by cvlearn913** [comment]
@yiheng-wang-nv Thanks for your reply！ I wrote a new function based on `normal_init`
`def kaiming_normal_init(
    m,  normal_func: Callable[[torch.Tensor, float, float], Any] = torch.nn.init.kaiming_normal_
) -> None:

    cname = m.__class__.__name__

    if getattr(m, "weight", None) is not None and (cname.find("Conv") != -1 or cname.find("Linear") != -1):
        normal_func(m.weight.data)
        if getattr(m, "bias", None) is not None:
            nn.init.constant_(m.bias.data, 0.0)

    elif cname.find("BatchNorm") != -1:
        normal_func(m.weight.data)
        nn.init.constant_(m.bias.data, 0)`
